IFC4x3/Sections/Resource definition data schemas/Schemas/IfcGeometricConstraintResource/Entities/IfcObjectPlacement/DocEntity.xml

```diff
@@ -11,6 +11,9 @@
 		<DocSubtype UniqueId="8200edef-bcea-4517-9b59-fbb49bcf03c7" DefinedType="IfcLinearPlacement" />
 	</Subtypes>
 	<Attributes>
+		<DocAttribute Name="PlacementRelTo" UniqueId="7bb09cb1-18c7-4ad5-a88d-f8ecccc28539" DefinedType="IfcObjectPlacement" AttributeFlags="1">
+			<Documentation>Reference to object placement that provides the relative placement with its placement in a grid, local coordinate system or linear referenced placement. If it is omitted, then in the case of linear placement it is established by the origin of horizontal alignment of the referenced IfcAlignment Axis. In the case of local placement it is established by the geometric representation context.</Documentation>
```

Comment:
I see that this is a verbatim copy of original text. However, i believe we need to rework this. E.g. IfcAlignment does not have Axis attribute. Should we do it in this pr or in a separate one?

Reply:
Good catch. Should do here since the relevant people are tagged.

Reply:
good documentation - should go into IFC4x3 spec after restoring the accidently deleted inverse

Reply:
@TLiebich the attribute is already defined in UML http://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/lexical/IfcObjectPlacement.htm#8.7.3.15.3-Attributes

Reply:
@aothms I think @TLiebich was talking about description. That is not available in the documentation yet apparently. It says  "No description available."